Dockerfile

```diff
@@ -33,6 +41,11 @@ RUN git clone https://github.com/NVIDIA/apex && \
     cd .. && \
     rm -rf apex
 
+# install warp-transducer
+ENV CXX=/usr/bin/g++-6
+ENV CC=/usr/bin/gcc-6
+RUN make deps/warp-transducer
```

Comment:
It can but in the docs it states that:
> The Dockerfile installs NVIDIA Apex, used for mixed precision, using a Python-only build and will omit some Apex features and performance improvements.

and I assumed that this was because CI doesn't like the proper apex build. 

This might have been the wrong assumption - should we be using the cpp build for CI?